tests/dataprep/test_dataprep_redis_finance.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
```

Comment:
This test script use gaudi, please rename it as `test_dataprep_redis_finance_on_intel_hpu.sh`.